libs/core/src/lib/multi-input/multi-input.component.ts

```diff
@@ -219,6 +218,23 @@ export class MultiInputComponent implements
     @Input()
     autoComplete = true;
 
+    /**
+     * Message announced by screen reader, when search suggestions opens.
+     */
+    @Input()
+    searchSuggestionMessage = 'suggestion found';
+
+    /**
+     * Second part of message for search suggestion.
+     * direction for navigating the suggestion. This is not necessry in case of 0 suggestion.
+     */
+    @Input()
+    searchSuggestionNavigateMessage = 'use up and down arrows to navigate';
+
+    /** Message for allowing token from values available */
+    @Input()
+    tokenSelectMessage = 'select value from the list';
+
```

Comment:
These being set by default, how would they be translated in different languages?

Reply:
using `localize` now. it is generating message for translation now.

<img width="1241" alt="Screenshot 2021-08-10 at 12 34 11 PM" src="https://user-images.githubusercontent.com/57661487/128825748-0cdd1308-5dff-49f5-8926-a143afc25b6a.png">

Reply:
following the final discussion. All these value should be a user input.